security/sshdo/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        raforg sshdo 1.1.1 v
+revision            0
+
+categories          security
+license             GPL-2+
+maintainers         {raf.org:raf @macportsraf} openmaintainer
+description         controls which commands may be executed via incoming ssh
+long_description    {*}${description}. \
+                    \n\nsshdo provides an easily configurable way of controlling which \
+                    commands may be executed via incoming ssh connections by specific \
+                    users and specific keys. Training mode allows a key to be used for \
+                    any command, while logging them. Then sshdo can learn from the logs \
+                    to know what commands to allow. It can also unlearn commands that \
+                    are no longer in use. This mitigates against private ssh key compromise \
+                    with very little effort.
+# Any of Python 2.6, 2.7, or 3.3+ will do.
+# When updating the python3 version, update the corresponding patch below as well.
+# Update the python3 version every year (or until there's a meta port:python3).
+depends_lib-append  port:python311
+
+homepage            https://raf.org/sshdo/
+github.tarball_from releases
+checksums           rmd160 3071b9aa06452437f9da970f0576c94ea27d3c08 \
+                    sha256 9b22e14fec786b692e8d6431a81e9b17ad8f6e62ae5d6ef9e9c643690bf33f90 \
+                    size 79485
+
+# Replace "python3" with the specific version depended-on above.
+# This means the user doesn't have to select a default python3 version.
+patchfiles          macports.patch
+
+# Note: Not a GNU autoconf configure script
+use_configure       yes
+configure.args      --macports-prefix=${prefix}
+build {}
```

Comment:
Same comment regarding the disabling of the build phase here as I left for the [danectl PR](https://github.com/macports/macports-ports/pull/19124).

Reply:
Thanks. I've added `supported_archs noarch`. But I haven't added `platforms any` because the installed files are different between macports/darwin and other BSD platforms (i.e., references in the installed files to /usr/local on FreeBSD and OpenBSD and references to /usr/pkg on NetBSD, and the references to the python executable. I think the default `darwin` platform is correct here. Does that sound right?

Reply:
Are there any differences in what the port installs for the different macOS versions? If not, then you should use:
```
platforms {darwin any}
```

Reply:
Thanks.

There are no differences between different versions of macOS, just differences between
different platforms. I've changed it to `{darwin any}`.

I had remembered this from the macports guide:

	The special value any can also be used to indicate that a port will
	install identical files across platforms or platform versions

But not this:

    Ports that install identical files on any Darwin version, but may
	install different files on other platforms, should use:
    platforms           {darwin any}